sdk/RELEASE.md

```diff
@@ -9,6 +9,99 @@
 ## Bug fixes and other changes
 
 ## Documentation updates
+# 2.0.0
+KFP SDK 2.0.0 release notes are distilled to emphasize high-level improvements in major version 2. See preceding 2.x.x pre-release notes for a more comprehensive list.
+
+Also see the KFP SDK v1 to v2 [migration guide](https://www.kubeflow.org/docs/components/pipelines/v2/migration/).
+## Features
+
+The KFP SDK 2.0.0 release contains features present in the KFP SDK v1's v2 namespace along with considerable additional functionality. A selection of these features include:
+* An improved and unified Python-based authoring experience for [components](https://www.kubeflow.org/docs/components/pipelines/v2/components/) and [pipelines](https://www.kubeflow.org/docs/components/pipelines/v2/pipelines/)
+* Support for using [pipelines as components](https://www.kubeflow.org/docs/components/pipelines/v2/pipelines/pipeline-basics/#pipelines-as-components) (pipeline in pipeline)
+* Various additional [configurations for tasks](https://www.kubeflow.org/docs/components/pipelines/v2/pipelines/pipeline-basics/#task-configurations)
+* Support for [fanning-in outputs](https://www.kubeflow.org/docs/components/pipelines/v2/pipelines/control-flow/#parallel-looping-dslparallelfor) from `dsl.ParallelFor` using `dsl.Collected`
+* Support for [obtaining upstream task status](https://www.kubeflow.org/docs/components/pipelines/v2/pipelines/control-flow/#exit-handling-dslexithandler) in a `dsl.ExitHandler` using `dsl.PipelineTaskFinalStatus`
```

Comment:
`dsl.PipelineTaskFinalStatus` doesn't feel to be a major feature worth calling out here, also because it's not ready on KFP backend yet. Suggest removing this. 
And arguably for `dsl.Collected` as well.

Reply:
Done.